user/vis/template.py

```diff
@@ -0,0 +1,48 @@
+pkgname = "vis"
+pkgver = "0.9"
```

Comment:
I intentionally packaged a git snapshot because there has been significant QoL changes in vis, and the last release has been from a long time ago.